debezium-event-api/pom.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
```

Comment:
@rareddy is already working on a Wildfly service, and will be refactoring it to reuse this new container engine pretty soon.